cddl-1-1-update.abnf

```diff
@@ -76,11 +76,14 @@ SCHAR = %x20-21 / %x23-5B / %x5D-7E / NONASCII / SESC
 SESC = "\" ( %x22 / "/" / "\" /                 ; \" \/ \\
              %x62 / %x66 / %x6E / %x72 / %x74 / ; \b \f \n \r \t
              (%x75 hexchar) )                   ; \uXXXX
-hexchar = non-surrogate / (high-surrogate "\" %x75 low-surrogate)
+
+hexchar = "{" (1*"0" [ hexscalar ] / hexscalar) "}" /
+          non-surrogate / (high-surrogate "\" %x75 low-surrogate)
```

Comment:
I don't like this backslash syntax, but it looks like it's always been there.

Reply:
We copied that from JSON...
(Not sure what specifically you don't like.)

Reply:
Sorry, I get it now.  The %75 bit obfuscated it slightly.  It's fine as-is.